src/main/java/teammates/ui/controller/InstructorFeedbackEditPageData.java

```diff
@@ -309,6 +309,15 @@ public void setStatusForAjax(String statusForAjax) {
         this.statusForAjax = statusForAjax;
     }
 
+    /**
+     * Retrieves the link to submit the request for copy of session. 
+     * Also contains feedback page link to return after the action.
+     * @return form submit action link
+     */
+    public String getEditCopyActionLink() {
+        return editCopyActionLink;
```

Comment:
I wonder if it will be better if we just returned <code>getInstructorFeedbackEditCopyActionLink</code>, instead of making new field?